docs/tuf-spec.txt

```diff
@@ -1108,10 +1108,19 @@ Version 1.0 (Draft)
    required, the root.json file is versioned and accessible by version number,
    e.g., 3.root.json. Clients update the set of trusted root keys by requesting
    the current root.json and all previous root.json versions, until one is
-   found that has been signed by keys the client already trusts. This is to
-   ensure that outdated clients remain able to update, without requiring all
-   previous root keys to be kept to sign new root.json metadata.
-
+   found that has been signed by a threshold of keys that the client already
+   trusts. This is to ensure that outdated clients remain able to update,
+   without requiring all previous root keys to be kept to sign new root.json
+   metadata.
+   
+   In the event that the keys being updated are root keys, it is important to
```

Comment:
I think we should.  Do we also explain how to recover from a fast-forward attack?

Note: I just reviewed section 7.1 and it seems accurate to me, at least in regard to filenames.  Is there a sentence/statement in particular that is inaccurate?